frontend/next.config.js

```diff
@@ -12,6 +12,11 @@ const withTM = require('next-transpile-modules')([
 
 const { withSentryConfig } = require('@sentry/nextjs');
 
+const withBundleAnalyzer =
+   process.env !== 'production'
```

Comment:
`process.env` is an object here.

Reply:
Fixed in [f1cd17f](https://github.com/iFixit/react-commerce/pull/347/commits/f1cd17f2e668cebed017dfac326ae1add49137cb)